UNRELEASED.md

```diff
@@ -0,0 +1,7 @@
+**Added:**
+
+- bpk-component-price
+  - Price is a new component.
+  - `title` is a required attribute to show price.
+  - `subtitle` is a optional attributes to show original price before the discount.
+  - `description` is a optional attribute which could be used to show the price is for how many nights.
```

Comment:
Since this is a new one, I think we can just add something to describe what the component is for, find the example [here](https://github.com/Skyscanner/backpack/blob/main/CHANGELOG.md)